packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -256,11 +255,6 @@ IEventProvider<IFluidDataStoreContextEvents>, Partial<IProvideFluidDataStoreRegi
     readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
     readonly storage: IDocumentStorageService;
     readonly baseSnapshot: ISnapshotTree | undefined;
-    /**
-     * @deprecated 0.37 Use the provideScopeLoader flag to make the loader
```

Comment:
I think 0.37 here means you can't merge it right away, likely want to give it 2 releases to bake?

Reply:
yup will wait a while before merging